spec/coverage_statistics_spec.rb

```diff
@@ -16,20 +16,20 @@
     end
 
     it "can omit the total strength defaulting to 0.0" do
-      statistics = described_class.new(covered: 4, missed: 6, total_strength: 0.0)
+      statistics = described_class.new(covered: 4, missed: 6)
 
       expect(statistics.strength).to eq 0.0
     end
 
     it "can deal with it if everything is 0" do
-      statistics = described_class.new(covered: 0, missed: 0, total_strength: 0.0)
+      statistics = empty_statistics
```

Comment:
technically yes, while the test description says "everything is 0" I like to see that everything is 0 that makes it immedeatly a bit clearer (for this one explicit test) and it's not such a huge amount of duplication imo.